apps/sms/style/sms.css

```diff
@@ -933,16 +932,29 @@ body[role="application"] form[role="dialog"][data-type="action"] > header:first-
 }
 
 /* Phone details label styles */
-.phone-type,
+.carrier-information {
+  display: flex;
+  flex-flow: row nowrap; 
+}
+
+.phone-type, .phone-carrier {
+  min-width: 0px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  flex: 0 1 auto;
+}
+
+.phone-number {
+  flex: 1 0 auto;
```

Comment:
you'll need to add a rule for phone-carrier-separator, with a min-width of 0.3em or 0.2em (try one or the other and pick the one that look the more like what we have today). Maybe `flex: 0 0 0.3em` (or 0.2em) works too, I don't really mind which one you pick.